lambdas/data-migration2/README.md

```diff
@@ -43,4 +43,6 @@ Migration summary:
         $ 0 records failed
 ```
 
-Records skipped indicates if a record was already migrated (which the lambda will throw with a `RecordAlreadyMigrated` error.)
\ No newline at end of file
+Records skipped indicates if a record was already migrated (which the lambda will throw with a `RecordAlreadyMigrated` error.)
```

Comment:
FWIW the above report summary structure is no longer accurate based on your previous refactors. To be honest I'm not sure if it's necessary to include a preview of the lambda output in the README

Reply:
In fact a lot of the README above is now inaccurate based on how this Lambda gets run. We don't have to clean it up in this PR necessarily, but need to make sure that we don't forget

Reply:
That's a good point. As for the lambda output, why do you think it's unnecessary to not include a preview of the output? I don't have strong views either way.

Reply:
Just because it seems like documentation that gets easily out of date and I'm not sure that it's the job of a README to give a preview of Lambda output. I see the README as describing mostly what something is and how to use it, if that is even applicable

Reply:
[`c6c1f32` (#2179)](https://github.com/nasa/cumulus/pull/2179/commits/c6c1f32eff6ceb5882a6dbb716014d89e99b8866#diff-4c92a92285abf37daad11f4a41b7ac4bf43e8649a39a04347600c035825cd2e9R20)